packages/api-explorer/src/components/MainWrapper/MainWrapper.tsx

```diff
@@ -1,10 +0,0 @@
-import React, { FC } from 'react'
```

Comment:
This was replaced with https://github.com/looker-open-source/sdk-codegen/pull/254/files#diff-0f5e466453ccc716b22495e3cdac3bcfR119-R121